src/common/systems/MineController.hpp

```diff
@@ -16,6 +16,9 @@ class MineController {
     uint32_t createMine(sf::Vector2f center_position);
     void update();
     Mine& getMine(uint32_t mine_id);
+    std::vector<std::shared_ptr<Mine>>& getMines() {
```

Comment:
We dont need this method considering we have getMines in GameObjectController
https://github.com/SourenP/ungroup_game/blob/4e387552297a774f777d2d2a549148cbffc172d3/src/common/systems/GameObjectStore.cpp#L59

Reply:
you linked GameObjectStore